docs/resources/mlplatform_jupyterhub.md

```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "ML Platform"
+layout: "vkcs"
+page_title: "vkcs: vkcs_mlplatform_jupyterhub"
+description: |-
+  Manages a ML Platform JupyterHub instance resource within VKCS.
+---
+
+# vkcs_mlplatform_jupyterhub
+
+Manages a ML Platform JupyterHub resource.
+
+## Example Usage
+```terraform
+resource "vkcs_mlplatform_jupyterhub" "jupyterhub" {
+    name = "tf-example"
+    admin_name = "admin"
+    admin_password = "dM8Ao21,0S264iZp"
```

Comment:
does ml require strict password or we can use here just "secret"

Reply:
Some validation is present, change sample password to "Password!"